tldw_Server_API/app/core/Evaluations/cli/evals_cli_enhanced.py

```
@@ -76,6 +83,27 @@
+    from tldw_Server_API.app.core.AuthNZ.User_DB_Handling import get_single_user_instance
+
+    return get_single_user_instance().id_str
+
+
+def _get_recipe_runs_service(*, user_id: str | None = None, db_path: str | None = None):
+    if db_path:
+        return RecipeRunsService(db=EvaluationsDatabase(db_path), user_id=user_id or _default_recipe_user_id())
+    return get_recipe_runs_service_for_user(user_id or _default_recipe_user_id())
+
+
+def _load_json_option(value: str | None, *, option_name: str, default: Any) -> Any:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `default`

(ANN401)